sample/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation deps.appcompatV7
     implementation deps.constraintLayout
     implementation deps.kotlin
+    implementation deps.supportV4
```

Comment:
No need, we should be getting the dependency transitively (or as suggested earlier wrap the `ResourcesCompat` call so that we don't need to expose it). Thinking of `sample` as an example project using Paris, the goal is that it not need to depend on anything other than Paris to use it.